x-pack/plugin/deprecation/src/main/java/org/elasticsearch/xpack/deprecation/IndexDeprecationChecks.java

```diff
@@ -257,7 +301,53 @@ static DeprecationIssue chainedMultiFieldsCheck(IndexMetadata indexMetadata) {
         return null;
     }
 
-    private static boolean containsChainedMultiFields(Map<?, ?> property) {
+    static DeprecationIssue chainedMultiFieldsDynamicTemplateCheck(IndexMetadata indexMetadata) {
```

Comment:
I don't see any test coverage of this method. Was that intentional (there's definitely a lot of code reuse with the ClusterDeprecationChecks methods)?

Reply:
added